incubator/ado-agent/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
I see the repo is tagged with the version 1.1. You might want to update the `appVersion` to the latest tag.